### ISSUE TITLE
Problem with png images

### ISSUE THREAD

**Issue by tanger1n** [haegul/node-image-filter#9]
I've tried use ur library for change brightness on image. For jpeg images it's work fine. But for png image it return { data: undefined, type: 'png', width: 285, height: 177
I don't understand why.
Thanks

**Comment by hg-pyun** [comment]
@komarevtsevdn 
Hello, Thank you for report. ASAP, I will check it.

**Comment by hg-pyun** [comment]
@komarevtsevdn 
I fixed this Issue #10 , you should update node-image-filter version to 0.1.0
```
$ npm install node-image-filter@0.1.0
``` 
I improved save image logic. So you must change your code like this.
```javascript
result.data.pipe(fs.createWriteStream(`result.${result.type}`)); // save local
```

Thank you.
